sufia-models/app/models/datastreams/fits_datastream.rb

```diff
@@ -144,42 +144,7 @@ def self.xml_template
     http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd",
                version: "0.6.0",
                timestamp: "1/25/12 11:04 AM") {
-        xml.identification {
```

Comment:
The intended purpose of this change is to get rid of the blank strings (`[""]` and `""`) from `GenericFiles` on initialization that are put into some attributes because of the content of this template. You can see what I mean by running `::GnericFile.new` (in current `master`) and looking at the content of the characterization related attributes. As I've indicated above, I'm not sure about the correctness of my solution or its backward compatibility. I'm not even sure what you mean by `we should deprecate all these methods`? I'm new to the project so you'll have to bear with me.